### PR TITLE
Choose between standard and prettier (#29)

### PR DESCRIPTION
**Pull request by pablopunk**
I got it working both with `standard` and `xo/prettier`. I had to do a manual `--fix` in sao's `post` to format the code depending on the linter.

Also, I think we would have some conflicts with https://github.com/lassjs/lass/pull/9, as both do similar things.

### CODE REVIEW

File: sao.js [hunk 1]
```diff
@@ -189,6 +197,13 @@ module.exports = {
     }
     */
 
+    // Format code according to eslint configuration
+    const linter = ctx.answers.eslint === 'standard' ? 'standard' : 'xo';
+    await spawn(`./node_modules/.bin/${linter}`, ['--fix'], {
```
Comment (OmgImAlexis) [inline sao.js]: I'd switch this to `execa`.

Comment (OmgImAlexis) [reply sao.js]: Is there a reason you're not just use `eslint` or `xo` for both fixes since standard is just a eslint config?

Comment (pablopunk) [reply sao.js]: @OmgImAlexis I was about to install `execa` but I realized I could just use the native `child_process`. But I can totally do that 👌🏻

**Comment by niftylettuce** [comment]
is this good to go? @pablopunk @OmgImAlexis

**Comment by pablopunk** [comment]
@niftylettuce I think so

**Pull request merged**